sites-config/site-stream.json

```diff
@@ -0,0 +1,25 @@
+{
+    "$id": "site-entity",
+    "filter": {
+      "entityIds": [
+        "turtlehead-tacos-site"
```

Comment:
Things brings up a question as to what the starter should contain. This requires a specific entity in your account to work, but most accounts won't have this entity unless they are following some prescribed instructions. 

This is a Product call. My opinion would be to _not_ prescribe anything so it'd work for any dev using it.

Reply:
agreed - I removed it.